CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### New features
 
+* Added `template-aliases.empty_placeholder`.
+
```

Comment:
Can this be expanded to describe what this can be used for?

Reply:
Done.